src/schemas/content.schema.ts

```diff
@@ -81,6 +81,16 @@ export class content {
 
   @Prop({ default: now() })
   updatedAt: Date;
+
+  @Prop({ type: String, required: true })
+  @IsOptional()
+  @IsString()
+  createdBy: string;
```

Comment:
@deepali-chavhan @gouravmore 
We have already field name as publisher. Do we need this fiels?